Dockerfile

```diff
@@ -1,66 +1,58 @@
-############################################################
-## Dockerfile to build the ga4gh server on mod_wsgi-express
-## Configurable to use a local dataset
-############################################################
-FROM ubuntu
+# Using multi stage to prevent keeping a second copy of the package in the image
+FROM centos:7.6.1810
+RUN yum -y install epel-release
+RUN yum -y install python36-pip.noarch \
+ git \
+ libffi-devel.x86_64 gcc-c++.x86_64 \
+ python36-devel.x86_64 openssl-devel \
+ libxml2-devel.x86_64 libxslt-devel.x86_64  libcurl-devel.x86_64 make gcc  \
+ && pip3 install --upgrade pip setuptools
 
-# Originally created by Steve Hershman GitHub @hershman
-# previously maintained by Alastair Firth, and Maciek Smuga-Otto of the
-# UCSC Genomics Institute
-MAINTAINER David Steinberg <david@resium.com>
+ENV SCHEMA_V=v1.0.0 INGEST_V=v1.3.0
 
-# Update the sources list
-RUN apt-get update  --fix-missing
-RUN apt-get upgrade --yes
+RUN  pip install \
+  git+https://github.com/CanDIG/candig-schemas.git@${SHEMA_V}#egg=candig_schemas  \
+  git+https://github.com/CanDIG/candig-ingest.git@${INGEST_V}#egg=candig_ingest \
+  gevent
 
-# Install packages
-RUN apt-get install -y tar git curl libcurl4-openssl-dev wget dialog \
-    net-tools build-essential python python-dev python-distribute \
-    python-pip zlib1g-dev apache2 libapache2-mod-wsgi libxslt1-dev \
-    libffi-dev libssl-dev
+COPY . /tmp/server
+RUN cd /tmp/server/ && pip install .
 
-# Enable wsgi module
-RUN a2enmod wsgi
+RUN mkdir /data
+WORKDIR /data
+RUN mkdir  mkdir candig-example-data \
+  && touch access_list.tsv
 
-# Create cache directories
-RUN mkdir /var/cache/apache2/python-egg-cache && \
-    chown www-data:www-data /var/cache/apache2/python-egg-cache/
+RUN curl -Lo /tmp/mock_data.json  https://github.com/CanDIG/candig-ingest/releases/download/${INGEST_V}/mock_data.json \
+ && ingest candig-example-data/registry.db mock_data /tmp/mock_data.json
 
-# build the GA4GH server
-RUN mkdir -p /srv/ga4gh/server
-WORKDIR /srv/ga4gh/server
 
-# Configure the python requirements
-# Do this as a separate step prior to the build so that changes
-# to the GA4GH Server codebase do not trigger a full rebuild of the
-# pip requirements.
-COPY requirements.txt /srv/ga4gh/server/
-COPY constraints.txt /srv/ga4gh/server/
-RUN pip install -r requirements.txt -c constraints.txt
+FROM centos:7.6.1810
+MAINTAINER P-O Quirion <pierre-olivier.quirion@calculquebec.ca>
+RUN yum -y install epel-release
+RUN yum -y install  \
+ gcc-c++.x86_64 \
```

Comment:
What's the gcc-c++ needed for here?

Reply:
For numpy I think. It is in the dependencies.

Reply:
It it numpy? I can check, but all the package there where needed for the build not to fail

Reply:
Actually a am going over that right now and we might not need c++ for the run time part of the build, but it depends on how the libs are compiled... and we do not control that.